packages/react-components/react-popover/src/components/PopoverTrigger/PopoverTrigger.types.ts

```diff
@@ -16,9 +16,4 @@ export type PopoverTriggerState = {
   children: React.ReactElement | null;
 };
 
-export type PopoverTriggerChildProps = {
-  ref?: React.Ref<never>;
-} & Pick<
-  React.HTMLAttributes<HTMLElement>,
-  'aria-expanded' | 'onClick' | 'onContextMenu' | 'onKeyDown' | 'onMouseEnter' | 'onMouseLeave'
->;
+export type PopoverTriggerChildProps = React.HTMLAttributes<HTMLElement>;
```

Comment:
Generalizes type for `TriggerChildProps` as we're using `useARIAButton` internally which means properties types are not 100% certain to be there, it might depend on the type of the `children` being passed to the `Trigger`